plaid/Cargo.toml

```diff
@@ -38,6 +38,7 @@ warp = { version = "0.3", features = ["tls"] }
 wasmer = { version = "4", default-features = false, features = ["cranelift"] }
 wasmer-middlewares = "4"
 jsonwebtoken = { version = "9.2" }
+sha3 = "0.10.8"
```

Comment:
Why import SHA3 for this? You could do a SHA2 from Ring without adding a new dependency. Is there a reason you want to pull in a new dep for this?